addons/website_blog/views/website_blog_posts_loop.xml

```diff
@@ -0,0 +1,250 @@
+
+    <a t-attf-href="/blog/#{slug(blog_post.blog_id)}/post/#{slug(blog_post)}"
+       t-attf-class="text-decoration-none d-block #{classes or 'mb-2'}"
+       t-att-style="'opacity:0.6;' if not blog_post.website_published else ''">
+
+        <t t-call="website_blog.blog_cover">
+            <t t-set="cover_properties" t-value="blog_posts_cover_properties[blog_post_index]"/>
+            <t t-set="additionnal_classes" t-value="'o_list_cover' + (' rounded overflow-hidden shadow mb-3' if not opt_blog_cards_design else '' )"/>
+
+            <t t-if="opt_posts_list_show_author" t-call="website_blog.post_author">
+                <t t-set="additionnal_classes" t-value="'o_wblog_post_list_author o_list_cover d-flex text-white w-100 ' + ('p-3 h5 m-0' if opt_blog_list_view else 'px-2 pb-2 pt-3') "/>
+                <t t-set="hide_date" t-value="True"/>
+            </t>
+        </t>
+    </a>
+</template>
+
+<!-- ======   Sub-Template: Posts list : Posts Excerpt + Tags  ============= -->
+<template id="website_blog.post_excerpt">
+    <a t-attf-href="/blog/#{slug(blog_post.blog_id)}/post/#{slug(blog_post)}" class="text-reset" t-att-title="blog_post.name">
+        <div t-if="opt_blog_list_view" t-field="blog_post.teaser" class="mt-2 o_wblog_read_text"/>
+        <div t-else="" t-field="blog_post.teaser_short" t-attf-class="mt-2 #{'o_wblog_normalize_font' if not opt_blog_readable else ''}"/>
+    </a>
+
+    <!-- Tags -->
+    <div t-if="len(blog_post.tag_ids)" class="o_wblog_post_short_tag_section d-flex align-items-center flex-wrap pt-2">
+        <t t-foreach="blog_post.tag_ids" t-as="one_tag">
+            <a t-attf-href="#{blog_url(blog=blog_post.blog_id, tag=tags_list(active_tag_ids, one_tag.id))}"
+               t-attf-class="badge mb-2 mr-1 #{one_tag.id in active_tag_ids and 'badge-primary' or 'border'}"
+               t-esc="one_tag.name"/>
+        </t>
+    </div>
+</template>
+
+
+<!--   ======================      OPTIONS      ===========================  -->
+<!--   ====================================================================  -->
+<!-- (Option) Posts List: Show Covers -->
+<template id="opt_posts_list_show_cover" name="Cover" inherit_id="website_blog.posts_loop" active="True" customize_show="True">
+    <xpath expr="//div/div" position="before">
+        <t t-set="opt_posts_list_show_cover" t-value="true"/>
+    </xpath>
+</template>
+
+<!-- (Option) Posts List: Show Author -->
+<template id="opt_posts_list_show_author" name="Author" inherit_id="website_blog.posts_loop" active="True" customize_show="True">
+    <xpath expr="//div/div" position="before">
+        <t t-set="opt_posts_list_show_author" t-value="true"/>
+    </xpath>
+</template>
+
+<!-- (Option) Posts List: Show Post Stats -->
+<template id="opt_posts_list_show_post_stats" name="Comments/Views Stats" inherit_id="website_blog.posts_loop" active="False" customize_show="True">
+    <xpath expr="//div/div" position="before">
```

Comment:
Avoid all xpath like this, ok in worst case in internal custo, not in standard. You can add a `data-name`, a `data-xpath-id`, an `id` or whatever to identify those ;)